### DISCUSSION TITLE
availability on hub.docker.com of theiaide/theia:1.18

### DISCUSSION THREAD

**Discussion by dginhoux** [eclipse-theia/theia#10275] [Q&A] [answered]
Hi everyone, 

I'm a poweruser of this great tool.
Do you planned to build and publish a docker image for the v1.18 ?

Have a good day.

**Comment by vince-fugnitto** [comment, marked as answer]
@dginhoux thanks for the discussion, unfortunately the docker images part of [theia-apps](https://github.com/theia-ide/theia-apps) will no longer be maintained or published (please see the following [issue](https://github.com/theia-ide/theia-apps/issues/496)). Given that the images were only examples for you to reference in order to build and maintain your own image(s), you can still use them to do so.

**Comment by dginhoux** [comment]
Hi, 

Thank you for your response.
I'll read the new bluepint project and hope be able to understand and build my own image to maintain this great tool.
 
Have a good day.
